exonize/exonize.py

```diff
@@ -80,9 +87,19 @@ def main():
                           self_hit_threshold=args.self_hit_threshold,
                           batch_number=args.batch_number,
                           threads=args.threads,
-                          timeout_db=args.timeout_db)
+                          timeout_db=args.timeout_db,
+                          genome_pickled_file_path=args.genome_pickled_file_path,
+                          )
     exonize_obj.run_exonize_pipeline()
 
 
 if __name__ == '__main__':
+    PROFILE_PATH = '/home/arthur/experimental/exonize/cProfile_dump_stats'
```

Comment:
:)

Reply:
That was a branch :), I didn't open a pull request with this inside. This is work in progress.